lib/iris/__init__.py

```diff
@@ -221,15 +225,8 @@ def context(self, **kwargs):
         statement, the previous state is restored.
 
         For example::
-
-            with iris.FUTURE.context():
-                iris.FUTURE.cell_datetime_objects = True
-                # ... code which expects time objects
-
-        Or more concisely::
-
-            with iris.FUTURE.context(cell_datetime_objects=True):
-                # ... code which expects time objects
+            with iris.FUTURE.context(cell_datetime_objects=False):
```

Comment:
Not even sure if we should still have this example here, since we would get an exception if we tried to use it...

Reply:
It is an example about the Futures, not anything else. I can live with it.

Reply:
Quite. But we do need an example of how this thing should be used... or do we any more? We could just add the usage example back in when a new future comes along... in the future.